kalite/version.py

```diff
@@ -7,7 +7,7 @@
 # Must also be of the form N.N.N for internal use, where N is a non-negative integer
 MAJOR_VERSION = "0"
 MINOR_VERSION = "14"
-PATCH_VERSION = "0"
+PATCH_VERSION = "alpha7"
```

Comment:
I've explained why I'm doing this a lot of times, I hope that's clear to all that it's a temporary thing.